### DISCUSSION TITLE
Custom icons designs

### DISCUSSION THREAD

**Discussion by youle31** [UPBGE/upbge#1578] [Resources for UPBGE 0.3+]
TriVoxel MacOS icon:

quote from Trivoxel: I have updated the macOS icons to look like a native macOS app. The new icon makes UPBGE match the rest of the system.

What the new icon looks like:
![icon](https://user-images.githubusercontent.com/25536957/135809518-b4ba0941-25dd-4a42-b75e-7961fe527b3f.png)

Some other custom icons found on BA:

https://blenderartists.org/t/upbge-0-3-0-logo-designs/1236263

**Comment by commie-caden** [comment]
Please note: this does not change the icon in Windows or Linux. This only affects macOS.

**Comment by MelvinG24** [comment]
Good morning everybody,

I like this icon so much that I really wish I can see it as the official icon for the upcoming versions of UPBGE. I took the free time to convert this PNG to real icons compatible with **Win XP, Vista, 7, 8, & 10** and compatible with **macOSX 10.4 ... to 12.0.1**

**Windows Icon:**
[Win_UPBGE_icon.zip](https://github.com/UPBGE/upbge/files/7548177/Win_UPBGE_icon.zip)

**Macintosh Icon:**
[Darwin_UPBGE_icon.zip](https://github.com/UPBGE/upbge/files/7548178/Darwin_UPBGE_icon.zip)
